src/base/io/utilities/convertOldStyleModel.m

```diff
@@ -0,0 +1,109 @@
+function model = convertOldStyleModel(model)
```

Comment:
Not really.
I could put in a function if you want, though I think it woud be pretty artificial, as I have to take an existing model with "old" fields and move it to a new model, and check whether the fields indeed got moved/merged.